src/7-to-8/cheat-sheet.rst

```diff
@@ -0,0 +1,279 @@
+Pausing & Unpausing
+-------------------
+
+Tell a workflow not to submit any new jobs:
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7** & Rose 2019
+     - **Cylc8** (Rose 2)
+   * - ::
+
+         cylc hold <name>
+
+         cylc unhold <name>
+     - ::
+
+         cylc pause <name>
+
+         cylc play <name>
+
+Stopping
+--------
+
+Stop a running workflow::
+
+   cylc stop <name>
+
+Restarting
+----------
+
+Restart a stopped workflow and pick up where it left off:
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7**
+     - **Rose 2019**
+     - **Cylc8** (Rose 2)
+   * - ::
+
+         # no installation capibility
+         # restart from source
+         cylc restart <name>
+     - ::
+
+         # regular restart
+         rose suite-restart
+
+       Or alternatively::
+
+         # reinstall and restart
+         rose suite-run --restart
+     - ::
+
+         # optionally reinstall
+         cylc reinstall <name>
+
+         # restart
+         cylc play <name>
+
+Deleting
+--------
+
+Delete the workflow :term:`run directory` (leave source files untouched):
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7**
+     - **Rose 2019**
+     - **Cylc8** (Rose 2)
+   * - ::
+
+         rm -rf ~/cylc-run/<name>
+     - ::
+
+         rose suite-clean
+     - ::
+
+         cylc clean
+
+       Or alternatively::
+
+         rm -rf ~/cylc-run/<name>
+
+Scanning
+--------
+
+List all running workflows::
+
+   cylc scan
+
+View A Workflow's Configuration
+-------------------------------
+
+View the parsed workflow configuration:
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7**
+     - **Rose 2019**
+     - **Cylc8** (Rose 2)
+   * - ::
+
+         cylc get-config --sparse \
+             <name/path>
+     - ::
+
+         # install workflow
+         rose suite-run -l
+
+         # view installed config
+         cylc get-config --sparse \
+             <name/path>
+     - ::
+
+         cylc config --sparse \
+             <name/path>
+
+Opening User Interfaces
+-----------------------
+
+Opening the graphical user interface (GUI) or terminal user interface (TUI)
+for monitoring / controling running workflows:
+
+.. list-table::
+   :class: grid-table
+
+   * -
+     - **Cylc 7** & Rose 2019
+     - **Cylc8** (Rose 2)
+   * - Terminal
+     - ::
+
+         cylc monitor <name>
+     - ::
+
+         cylc tui <name>
+   * - Graphical
+     - ::
+
+         cylc gui <name>
+     - * Open the Cylc Hub's URL in a web browser.
+
+       * Or alternatively run your own hub::
+
+           cylc hub
+
+         Then open the address ``0.0.0.0:8000`` in a web browser.
+
+Static Graph Visualisation
+--------------------------
+
+Generate a visualisation for a workflow without running it:
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7** & Rose 2019
+     - **Cylc8** (Rose 2)
+   * - ::
+
+         cylc graph <name>
+     - Not yet implemnted. As a temporary workaround Graphviz can be used
```

Comment:
```suggestion
     - Not yet implemented. As a temporary workaround Graphviz can be used
```

Reply:
Also, this renders as a single very long line, with a nasty horizontal scroll bar.